src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://google.pl"
+      data-qa="big"
+    >
+      <label class="box">
```

Comment:
Add **for** attribute to label to point to the element it is attached to for better accessibility